thirdeye/thirdeye-frontend/app/pods/manage/alert/explore/template.hbs

```diff
@@ -157,11 +141,11 @@
 
       {{#if filteredAnomalies}}
         {{!-- Baseline type selector --}}
-        <div class="te-pill-selectors">
-          <div class="te-pill-selectors__label">Baseline</div>
-          <ul class="te-alert-page-hzlist">
+        <div class="te-pill-selectors__wrapper">
```

Comment:
TODO (follow-up PR): is it worth componentizing each `te-pills`?

Reply:
Instead, I am now using the same `range-pill-selectors` component to render these.